### PR TITLE
Decrease default learning rate.

### PR DESCRIPTION
**Pull request by deflaux**
Also:
* use CloudML Engine runtime version instead of overriding the TensorFlow pip package.
* update sample_has_variant to return an allele count that can also be interpreted as a boolean

### CODE REVIEW

**Review by mbookman — APPROVED** [review]
LGTM
